api/src/auth/strategies/jwt.strategy.ts

```diff
@@ -24,8 +24,9 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
     }
 
     async validate(jwtPayload: AuthJwtPayload) {
-        const user = await this.userService.findOne(jwtPayload.id);
-        if (!user) throw new UnauthorizedException();
+        await this.userService.findOne(jwtPayload.id).catch(() => {
+            throw new UnauthorizedException();
```

Comment:
Why error like this? When this occures something other that Unauthorized. Database error which sould be hanled with 500 or someting else You will never be aware what is this. User will be given with Unauthorized even when credential will be correct. Very bad UX and error handling.

Reply:
500 will occure, when something gone wrong on server, but here i can't return 404 Not Found, because then on client security is bad, on potential attack, or manual probe of login on somebody other account, user have info about "That account didn't exist", but if exist, then user have a username, and can try to break password.

Reply:
i can create a message for this exceptions

Reply:
Done, simplify all exceptions